nhais/sequence/interchange_id.py

```diff
@@ -0,0 +1,15 @@
+from sequence.sequence_factory import get_sequence_generator
+
+
+class InterchangeIdGenerator:
+    """A component that provides sequential interchange id.
+    Definition:
+    SIS (Send Interchange Sequence) - sequence number for the entire EDIFACT interchange """
+
+    def __init__(self):
+        self.table_name = 'transaction_id_counter'
+
+    async def generate_interchange_id(self, sender, recipient) -> int:
+        key = "SIS-" + str(sender) + "-" + str(recipient)
```

Comment:
better to use an fstring than + concatenation
f"SIS-{sender}-{recipient}"

Reply:
Thank you, good idea :)